### PR TITLE
Add urls for the link to the documentation page

### LINKED ISSUE #1780: Documentation for customer multi-currency
Add a new section about Multi Shopper Currencies to the docs, including:

- How this new feature works:
  - Onboarding process
  - Settings
  - Widget and block
  - Analytics
- Disclaimers about the currency conversion rates.
- Explaining the link between merchant account currency and shopper payments currency.

#### (TODOs) Links to documentation:
- `class-settings.php` `// TODO: Learn more documentation link, to be done on #1780`

### PR DESCRIPTION
**Pull request by dwainm**
Fixes #1780 

#### Changes proposed in this Pull Request
Add the URL to the docs link and remove inline TODO items.

#### Testing instructions

- Visit the multi-currency settings page and verify that the "Learn More" link at the top takes you to the documentation page.

*

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

**Review by dmallory42 — APPROVED** [review]
LGTM. I left one small suggestion to consider, but this is good to :shipit:.

File: includes/multi-currency/Settings.php
Comment (dmallory42) [inline includes/multi-currency/Settings.php:94]: Perhaps we should add the link to the docs as a `const` at the top of the file so that we can update it easily in one place if needed in future?

Comment (dwainm) [reply includes/multi-currency/Settings.php]: Thank for the review Daniel, added the change in 62518a38.

Comment (dmallory42) [reply includes/multi-currency/Settings.php]: Looks good, thanks for making that change! 

It looks like Psalm is complaining about something here: https://github.com/Automattic/woocommerce-payments/pull/2554/checks?check_run_id=3201195456

Once that is fixed, this looks good to merge to me! 👍

Comment (dwainm) [reply includes/multi-currency/Settings.php]: 🤦 I fixed the merge conflict on GH and left an empty line. Just fixed it up. Will merge after the builds are done.

**Pull request merged**
